templates/fullstack/client/src/graphql/graphback.graphql

```diff
@@ -0,0 +1,147 @@
+fragment NoteFields on Note {
```

Comment:
Users complained about generated queries being mixed with their own ones and their changes being overwritten by generator. After periods of trying things with detecting user changes in the files we decided to go with single graphql file that contains all queries as we always feed that into the GraphQL-Code-Generator. GraphQL-Code-Generator is now core for graphback - it gives hooks and resolver typing that are essential for the developers.